lib/core/src/lib/form/components/widgets/date-time/date-time.widget.ts

```diff
@@ -42,68 +43,115 @@ import { WidgetComponent } from '../widget.component';
+            this.field.value = newDate;
+            this.updateField();
+        });
+    }
 
-            if (this.field.maxValue) {
-                this.maxDate = DateFnsUtils.getDate(this.field.maxValue);
-            }
+    private validateField(): void {
+        if (this.datetimeInputControl?.invalid) {
+            this.handleErrors(this.datetimeInputControl.errors);
+            this.field.markAsInvalid();
+        } else {
+            this.resetErrors();
+            this.field.markAsValid();
+        }
+    }
 
-            if (this.field.value) {
-                this.value = DateFnsUtils.getDate(this.field.value);
+    private handleErrors(errors: ValidationErrors): void {
+        const errorAttributes = new Map<string, string>();
+        switch (true) {
+            case !!errors.matDatepickerParse:
+                this.updateValidationSummary(this.field.dateDisplayFormat || this.field.defaultDateTimeFormat);
+                break;
+            case !!errors.required:
+                this.updateValidationSummary('FORM.FIELD.REQUIRED');
+                break;
+            case !!errors.matDatepickerMin: {
+                const minValue = DateFnsUtils.formatDate(errors.matDatepickerMin.min, this.field.dateDisplayFormat).toLocaleUpperCase();
+                errorAttributes.set('minValue', minValue);
+                this.updateValidationSummary('FORM.FIELD.VALIDATOR.NOT_LESS_THAN', errorAttributes);
+                break;
+            }
+            case !!errors.matDatepickerMax: {
+                const maxValue = DateFnsUtils.formatDate(errors.matDatepickerMax.max, this.field.dateDisplayFormat).toLocaleUpperCase();
+                errorAttributes.set('maxValue', maxValue);
+                this.updateValidationSummary('FORM.FIELD.VALIDATOR.NOT_GREATER_THAN', errorAttributes);
+                break;
             }
+            default:
+                break;
         }
     }
 
-    onValueChanged(event: Event) {
-        const input = event.target as HTMLInputElement;
-        const newValue = this.dateTimeAdapter.parse(input.value, this.field.dateDisplayFormat);
+    private updateValidationSummary(message: string, attributes?: Map<string, string>): void {
+        this.field.validationSummary = new ErrorMessageModel({ message, attributes });
+    }
 
-        if (isValid(newValue)) {
-            this.field.value = newValue.toISOString();
-        } else {
-            this.field.value = input.value;
-        }
+    private resetErrors(): void {
+        this.updateValidationSummary('');
+    }
 
-        this.value = DateFnsUtils.getDate(this.field.value);
-        this.onFieldChanged(this.field);
+    private initDateAdapter(): void {
+        if (this.field?.dateDisplayFormat) {
+            const dateAdapter = this.dateAdapter as AdfDateFnsAdapter;
+            dateAdapter.displayFormat = this.field.dateDisplayFormat;
+
+            const dateTimeAdapter = this.dateTimeAdapter as AdfDateTimeFnsAdapter;
```

Comment:
Instead of type casting maybe you can define the type when you inject DateTimeAdapter?

Reply:
currently, when we add the type at the inject adapter level, we get this error:

`Type 'DateAdapter<any, any>' is missing the following properties from type 'AdfDateFnsAdapter': displayFormat, formatsts(2739)`
so for now, i would prefer to skip additional refactoring around these adapters, we can consider it as a future improvement. is it ok?
Issue -> https://hyland.atlassian.net/browse/AAE-24763